cl/opinion_page/tests.py

```diff
@@ -1,6 +1,13 @@
 from django.conf import settings
 from django.urls import reverse
 from django.test import TestCase, override_settings
+from django.test.client import Client
+from django.contrib.auth.models import User, Group
+from django.core.files.uploadedfile import SimpleUploadedFile
+
+from .forms import TennWorkersForm
+import datetime
```

Comment:
Usually this kind of an import should go at the top. If you use PyCharm's import tool or import cleanup tools, they'll do that for you.

Reply:
√